lib/crawl.js

```diff
@@ -1,16 +1,39 @@
 'use strict';
 
 const dirname = require('path').dirname;
+const fs = require('fs');
 const simplecrawler = require('simplecrawler');
 const ora = require('ora');
 const cliTruncate = require('cli-truncate');
 const chalk = require('chalk');
 const mkpath = require('mkpath');
 const jsonfile = require('jsonfile');
-const cheerio = require('cheerio')
-const defaultConf = require('./default-config');
+const cheerio = require('cheerio');
 const limitSimilar = require('./limit-similar');
 
+let defaultConf = {};
+
+const rootPath = process.cwd();
+
+/**
```

Comment:
This could be simplified to something like

```js
const confPath = path.resolve(process.cwd(), 'backstop-crawl.js');
const defaultConf = fs.existsSync(confPath) ? require(confPath) : require('./default-config');
```

Reply:
On second though, do you thiink if it's just json in the file it should just be a `.json` file without the module.exports?

Reply:
I updated to `backstop.template.json`